docs/content/documentation/getting-started/configuration/index.md

```diff
@@ -253,6 +253,12 @@ index_format = "elasticlunr_javascript"
 
 Zola currently has the following highlight themes available:
 
+{{ grid_gallery() }}
+
+<details>
```

Comment:
@Keats 
This line! https://github.com/getzola/zola/issues/2928#issuecomment-3188739833